collectors/common.py

```diff
@@ -74,24 +100,30 @@ def fread(file_name=None):
     """
     Simple read function for files of a single value
     :param file_name: (str) file name to read
-    :return: (str) contents of the file
+    :return: (str) contents of the file, or null string for empty file
```

Comment:
doc text updated. In the context of the use case the file contents either exist with data or don't exist - but I take your point.